spec/models/member_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Member do
+  let(:group) { Group.create!(name: 'Design') }
+  let(:member) { Member.create!(name: 'John Doe', email: 'jd@artsymail.com', group_ids: [group.id]) }
+  context '#email' do
+    it 'stores a lowercase version of #email' do
+      expect(member).to be_valid
+    end
+  end
+
+  context 'with an existing #email' do
+    let(:member1) { Member.create!(name: 'John Doe', email: member.email, group_ids: [group.id]) }
+    let(:member2) { Member.create!(name: 'John Doe', email: member.email.upcase, group_ids: [group.id]) }
+    it 'does not allow another member with same #email' do
+      expect do
+        member1
```

Comment:
Here're you're misusing a `let`, in this case you want to explicitly create a `Member`.